worker/src/ee/cloudSpendAlerts/handleCloudSpendAlertJob.ts

```diff
@@ -0,0 +1,258 @@
+import { parseDbOrg, Role } from "@langfuse/shared";
+import { prisma } from "@langfuse/shared/src/db";
+import Stripe from "stripe";
+import { env } from "../../env";
+import { logger } from "@langfuse/shared/src/server";
+import { recordIncrement, traceException } from "@langfuse/shared/src/server";
+import { Job } from "bullmq";
+import { backOff } from "exponential-backoff";
+import { sendCloudSpendAlertEmail } from "@langfuse/shared/src/server";
+
+export const handleCloudSpendAlertJob = async (job: Job<{ orgId: string }>) => {
+  const { orgId } = job.data;
+
+  logger.info(`[CLOUD SPEND ALERTS] Processing org ${orgId}`);
+
+  if (!env.STRIPE_SECRET_KEY) {
+    logger.warn("[CLOUD SPEND ALERTS] Stripe secret key not found");
+    throw new Error("Stripe secret key not found");
+  }
+
+  // Fetch organization with spend alerts
+  const orgData = await prisma.organization.findFirst({
+    where: {
+      id: orgId,
+    },
+    include: {
+      cloudSpendAlerts: true,
+    },
```

Comment:
Shall we already filter out "Triggered" alerts here?

Reply:
We need them to handle resetting after billing cycle switchovers